Source/RageCommunity.Library/Graphics/Checkpoint.cs

```diff
@@ -56,9 +60,19 @@ public class Checkpoint : IDeletable, ISpatial
         /// </summary>
         /// <remarks>
         /// Checkpoint type 47 for example is a simple cylinder with no arrow or number.
-        /// Visit the Grand Theft Auto V Native DB for a list of all checkpoint types. 
+        /// Visit the Grand Theft Auto V Native DB for a list of all checkpoint types.
+        /// https://nativedb.dotindustries.dev/natives?search=checkpoint
```

Comment:
Check out [this post on StackOverflow](https://stackoverflow.com/questions/6960426/c-sharp-xml-documentation-website-link) about integrating URLs.  Will this suggestion work?
```suggestion
        /// Visit the <a href="https://nativedb.dotindustries.dev/natives?search=checkpoint">Grand Theft Auto V Native DB</a> for a list of all checkpoint types.
```

Reply:
Cool stuff! I wasn't aware of the fact that html links do work in summaries.